RealmTasks Apple/RealmTasks Shared/SharedFunctionality.swift

```diff
@@ -70,29 +70,29 @@ private func setDefaultRealmConfigurationWithUser(user: SyncUser) {
 
 // returns true on success
 func configureDefaultRealm() -> Bool {
-    if let user = SyncUser.currentUser() {
-        setDefaultRealmConfigurationWithUser(user)
+    if let user = SyncUser.current {
+        setDefaultRealmConfigurationWithUser(user: user)
```

Comment:
IMO this function should be renamed so the call site is `setDefaultRealmConfiguration(user: user)`